Document `postgresql.include-system-tables` config property (#18574)
trinodb/trino: docs/src/main/sphinx/connector/postgresql.md

use {doc}`secrets </security/secrets>` to avoid actual values in the catalog
properties files.

### Access to system tables

> The PostgreSQL connector supports reading PostgreSQL system tables, such as `pg_catalog`.

Reviewer: Please wrap at 80

The functionality is disabled by default, and can be enabled using the ``postgresql.include-system-tables`` configuration property.

```

> SHOW SCHEMAS FROM pg_catalog;

Reviewer: sorry .. I am confused .. in what schemas are the system tables exposed when this is enabled? 

Also .. should the example not be

show tables from example.notsurewhatschema 

or 

describe example.notsurewhatschema.pg_catalog ?

```

> More details about the PostgreSQL system tables, see the [PostgreSQL system 

Reviewer: Empty line after code snippet

use {doc}`secrets </security/secrets>` to avoid actual values in the catalog
properties files.

### Access to system tables

The PostgreSQL connector supports reading PostgreSQL system tables, such as
`pg_catalog`. The functionality is disabled by default, and can be enabled using
the `postgresql.include-system-tables` configuration property.

You can see more details about system tables in the `pg_tables` table in the 

> pg_catalog database using either of the following:

Reviewer: ```suggestion
`pg_catalog` schema using either of the following:
```

```
SHOW TABLES FROM example.pg_catalog;
SELECT * FROM example.pg_catalog.pg_tables;
```

More details about the PostgreSQL system tables, see the [PostgreSQL system 

> catalog documentation](https://www.postgresql.org/docs/current/catalogs.html).

Reviewer: According to that URL there is no pg_tables table...

## Querying PostgreSQL

The PostgreSQL connector provides a schema for every PostgreSQL schema. 

> It also supports reading PostgreSQL system tables, e.g., `pg_catalog` relations.

Reviewer: ```suggestion
It also supports reading PostgreSQL system tables, such as `pg_catalog`.
```

> The functionality is disabled by default but this can be enabled using the ``postgresql.include-system-tables`` configuration property.

Reviewer: ```suggestion
The functionality is disabled by default, and can be enabled using the ``postgresql.include-system-tables`` configuration property.
```
